packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -226,18 +229,14 @@ export const ConnectedWalletDetails: React.FC<{
         }}
       >
         {/* Address */}
-        {addressOrENS ? (
-          <Text
-            size="xs"
-            color="primaryText"
-            weight={500}
-            className={`${TW_CONNECTED_WALLET}__address`}
-          >
-            {addressOrENS}
-          </Text>
-        ) : (
-          <Skeleton height={fontSize.xs} width="80px" />
```

Comment:
any reason you removed this?

Reply:
It seems like `addressOrENS` should always have a value, so I removed the condition + skeleton component when adding the `connectedWalletName` into the mix